packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -105,6 +105,12 @@ export class ContainerStorageAdapter implements IDocumentStorageService, IDispos
 		this.getBlobContents(snapshotTree);
```

Comment:
with my other suggestions, we should be able to remove this method, and the methods only it uses